### PR TITLE
Ldanzinger/create save map

### PR DESCRIPTION
**Pull request by ldanzinger**
@michael-tims please review/merge

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/Maps/CreateAndSaveMap/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Create and save map
+
+This sample demonstrates how to author a `Map` in ArcGIS Runtime and save it to ArcGIS Online as a web map.
+
+![](screenshot.png)
+
+## How to use the sample
+- Select the basemap and operational layers to include in the `Map`.
+- Press the save button to save the `Map`.
+- Sign into ArcGIS Online with a valid organizational account.
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Maps/CreateAndSaveMap/README.md]: Not certain, but I think it should be `Sign in to ArcGIS Online...`

File: ArcGISRuntimeSDKQt_QMLSamples/Maps/CreateAndSaveMap/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Create and save map
+
+This sample demonstrates how to author a `Map` in ArcGIS Runtime and save it to ArcGIS Online as a web map.
+
+![](screenshot.png)
+
+## How to use the sample
+- Select the basemap and operational layers to include in the `Map`.
+- Press the save button to save the `Map`.
+- Sign into ArcGIS Online with a valid organizational account.
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_QMLSamples/Maps/CreateAndSaveMap/README.md]: `Sign in to ArcGIS Online...`

File: ArcGISRuntimeSDKQt_QMLSamples/Maps/CreateAndSaveMap/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- Give the `Map` a title, and optionally give it a description and some tags.
+- Press the save button to save the `Map` to ArcGIS Online with the provided information.
+
+## How it works
+- A `Map` is created with the user options and added to a `MapView`.
+- A `Portal` object is created by setting the URL to `https://www.arcgis.com` and setting `loginRequired` to true.
+- Once the Save button is pressed, the `Portal` load cycle is started, which will trigger an `AuthenticationChallenge`.
+- User credentials are gathered and passed into the `AuthenticationManager` to satisfy the `AuthenticationChallenge`.
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_QMLSamples/Maps/CreateAndSaveMap/README.md]: Technically user credentials are passed into the challenge object via the AuthenticationView control.

**Comment by ldanzinger** [comment]
@michael-tims updated

**Pull request merged**